Tests/UnitTests/Screens/TokensViewControllerTests.swift

```diff
@@ -5,17 +5,22 @@ import XCTest
 final class TokensViewControllerTests: XCTestCase {
     var accessToken: String!
     var sut: TokensViewController!
+    var didTapDeveloperMenu: Bool!
     
     override func setUp() {
         super.setUp()
         
-        sut = TokensViewController()
+        didTapDeveloperMenu = false
+        sut = TokensViewController(TokensViewModel {
+            self.didTapDeveloperMenu = true
+        })
```

Comment:
Same to the other comment about initialising the view model and passing in above if you don't mind